tests/content-block-manager.spec.js

```diff
@@ -14,7 +14,7 @@ async function verifyUpdatedRateVisible(page, url, updatedRate) {
 
 test.describe("Content Block Manager", { tag: ["@app-content-block-manager"] }, () => {
   const contentBlockPath = `${publishingAppUrl("content-block-manager")}/18`;
-  const whitehallPath = `${publishingAppUrl("whitehall-admin")}/government/admin/standard-editions/1658299`;
+  const whitehallPath = `${publishingAppUrl("whitehall-admin")}/government/admin/editions/1658299`;
```

Comment:
Is it worth including a link to some info about the "new edition format"? I for one would be interested in learning about this...